src/common/HTTPRequest.ts

```diff
@@ -626,13 +629,13 @@ export class HTTPRequest {
   }
 
   private async _abort(
-    errorReason: Protocol.Network.ErrorReason
+    errorReason: Protocol.Network.ErrorReason | undefined
   ): Promise<void> {
     this._interceptionHandled = true;
     await this._client
       .send('Fetch.failRequest', {
         requestId: this._interceptionId,
-        errorReason,
+        errorReason: errorReason || 'Failed',
```

Comment:
I don't know what this client function does though this feels like a behavior change:
* Before, when `errorReason` was given as `undefined` we were sending this as `undefined`. Now we're adding a default value `Failed` to it, is it okay?

Reply:
I think so as `Fetch.failRequest` requires a reason and sending undefined would result in failure.

Reply:
I think it's alright to do that if there is a reason missing for some reason :) the abort() API also defaults to the failed status if no reason is provided. I suspect this does not make much difference in practice but I'd like to avoid introducing runtime errors for this.